FEWD/Week03/lesson06/homework/startup_matchmaker/style/style.css

```diff
@@ -0,0 +1,243 @@
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after {
+  clear: both;
+}
+
+/*HEADER, BODY, FOOTER OF PAGES*/
+
+body {
+  width: 100%;
+  font-family: "Source Sans Pro", helvetica, sans-serif;
+  color: #000;
+  font-size: 12px;
+  line-height: 1.4;
+}
+
+header {
+  color: white;
+  background-color: #000;
+  height: 75px;
+  border-bottom: 5px solid #ffff1a;
+}
+
+h1 {
+  font-size: 40px;
+  font-weight: 700;
+  margin-left: 25px;
+  float: left;
+  font-family: Merriweather;
+  padding-top: 5px;
+}
+
+nav {
+  float: right;
+  height: 75px;
+  padding: 20px 0;
+}
+
+
+header li {
+  display: inline-block;
+  font-size: 20px;
+  font-weight: 600;
+  padding-right: 25px;
+}
+
+.font a:hover {
+  background-color: #b3b3b3;
+  padding: 15px 10px;
+}
+
+header li a:hover {
+  background-color: #b3b3b3;
+  padding: 22px 10px 18px;
+}
+
+header li a {
+  text-decoration: none;
+  color: white;
+}
+
+.font {
+  font-size: 24px;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-family: oswald;
+}
+
+footer {
+  background-color: #000;
+  color: white;
+  font-size: 14px;
+}
+
+footer p {
+  padding: 10px;
+}
+
+/*END OF GENERIC STYLING*/
+
+/*INDEX HTML CSS*/
+.tag-line {
+  background-color: #cccccc;
+  padding: 10px;
+  font-size: 28px;
+  font-style: italic;
+  font-weight: 600;
+  text-align: center;
+  font-family: Merriweather;
+}
+
+.strikethrough {
+  text-decoration: line-through;
+  color: #b3b3b3;
+}
+
+.underline {
+  text-decoration: underline;
+  -webkit-text-decoration-color: #ffff1a;
+  text-decoration-color: #ffff1a;
```

Comment:
This was definitely one of the trickiest bits in the HW. So, while the intention was definitely for you guys to make use of `text-decoration` for the strikethrough, this was bit of a harder one. Yes, you could do what you're trying to do here but a couple of things: 
1) browsers aren't good at supporting `text-decoration` that's a different colour than the text itself
2) browsers aren't good at allowing separation between the text and the underline
3) browsers aren't good at letting you set a thicker underline than just 1px

So, all in all, it's better to use `border-bottom`. If you apply this CSS:

```
.underline {
  padding-bottom: 1px;
  border-bottom: 3px solid yellow;
}
```

... you get an effect that's like the one in the reference image:

![image](https://cloud.githubusercontent.com/assets/5670279/17487015/f6c3a66a-5d8a-11e6-8cc0-ffa5832beb7d.png)